### PR TITLE
bluetooth: tester: VCP Client tests

### PR DESCRIPTION
**Pull request by piotrnarajowski**
Adding support for VCP Client tests

### CODE REVIEW

File: tests/bluetooth/tester/src/btp/btp_vocs.h [hunk 1]
```diff
@@ -23,3 +24,31 @@ struct btp_vocs_audio_desc_cmd {
...
+struct btp_vocs_state_get_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_VOCS_LOCATION_GET			0x05
+struct btp_vocs_location_get_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_VOCS_OFFSET_STATE_SET			0x06
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp/btp_vocs.h:38]: Indentation
```suggestion
#define BTP_VOCS_OFFSET_STATE_SET		0x06
```

File: tests/bluetooth/tester/src/btp_vcp.c [hunk 2]
```diff
@@ -7,28 +7,61 @@
...
 #define BT_AICS_MAX_OUTPUT_DESCRIPTION_SIZE 16
 
 static struct bt_vcp_vol_rend_register_param vcp_register_param;
+static struct bt_vcp_vol_ctlr *vol_ctlr;
 static struct bt_vcp_included included;
 extern struct btp_aics_instance aics_server_instance;
+extern struct btp_aics_instance aics_client_instance;
+extern struct bt_aics_cb aics_client_cb;
+
+struct ServiceHandles {
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_vcp.c:39]:
```suggestion
struct service_handles {
```

File: tests/bluetooth/tester/src/btp_vcp.c [hunk 3]
```diff
@@ -378,3 +535,423 @@ uint8_t tester_unregister_vocs(void)
...
+		chrc_handles.vocs_handles.state = cli->state_handle;
+		chrc_handles.vocs_handles.location = cli->location_handle;
+		chrc_handles.vocs_handles.control = cli->control_handle;
+		chrc_handles.vocs_handles.desc = cli->desc_handle;
+		chrc_handles.aics_handles.state = vol_ctlr->aics[0]->cli.state_handle;
+		chrc_handles.aics_handles.gain = vol_ctlr->aics[0]->cli.gain_handle;
+		chrc_handles.aics_handles.type = vol_ctlr->aics[0]->cli.type_handle;
+		chrc_handles.aics_handles.status = vol_ctlr->aics[0]->cli.status_handle;
+		chrc_handles.aics_handles.control = vol_ctlr->aics[0]->cli.control_handle;
+		chrc_handles.aics_handles.desc = vol_ctlr->aics[0]->cli.desc_handle;
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_vcp.c:648]: How come vocs and AICS are being treated differently here?

**Review by Thalley — CHANGES_REQUESTED** [review]

File: tests/bluetooth/tester/src/btp/btp_vocs.h (hunk 1 above)
Comment (piotrnarajowski) [reply tests/bluetooth/tester/src/btp/btp_vocs.h]: Fixed

File: tests/bluetooth/tester/src/btp_vcp.c (hunk 2 above)
Comment (piotrnarajowski) [reply tests/bluetooth/tester/src/btp_vcp.c]: Fixed

File: tests/bluetooth/tester/src/btp_vcp.c (hunk 3 above)
Comment (piotrnarajowski) [reply tests/bluetooth/tester/src/btp_vcp.c]: Fixed

**Review by Thalley — DISMISSED** [review]

**Review by Thalley — APPROVED** [review]

**Review by larsgk — APPROVED** [review]

**Pull request merged**